client/stories/0-general/1-userRoles.stories.mdx

```diff
@@ -22,10 +22,11 @@ For more detailed information on users' duties, see
 ## Superusers
 
 In addition to everything the normal, unprivileged users can do, ANET superusers have a few extra
-capabilities. A superuser can for example create new locations and new principals.
+capabilities. A superuser can for example create new locations and new principals, as well as create
+new positions for principal organizations.
```

Comment:
```suggestion
capabilities. A superuser is able to create:
- New locations
-  New principals (interlocutors, persons)
- Creating new positions in principal organizations
```

Is he not able to edit organisations and positions?
Does this also apply to orgnizations he has not been assigned to?